x-pack/plugins/security_solution/server/lib/siem_migrations/rules/api/start.ts

```diff
@@ -59,7 +59,7 @@ export const registerSiemRuleMigrationsStartRoute = (
           const inferenceClient = ctx.securitySolution.getInferenceClient();
           const actionsClient = ctx.actions.getActionsClient();
           const soClient = ctx.core.savedObjects.client;
-          const rulesClient = ctx.alerting.getRulesClient();
+          const rulesClient = await ctx.alerting.getRulesClient();
```

Comment:
@semd you're migrations work shouldn't be affected, but just a heads up if you need to test anything here